grouper/models.py

```diff
@@ -253,6 +253,10 @@ def name(self):
     def type(self):
         return "User"
 
+    @property
+    def is_enabled(self):
+        return self.enabled
```

Comment:
More future-proof if we ever want to do more here, set this value from outside the class or change the attribute name.